server/src/main/java/org/elasticsearch/index/mapper/vectors/MultiDenseVectorScriptDocValues.java

```diff
@@ -17,7 +17,7 @@
 
 public class MultiDenseVectorScriptDocValues extends ScriptDocValues<BytesRef> {
```

Comment:
change these files names as well?

Reply:
This is something I am debating. The field is a `rank_vectors` but it provides access to a `MultiDenseVector` set of values. For example, `half_float` `float`, `double` all return a `Doubles` object. So, these script doc values aren't named after the field type, but instead the values they return.

Reply:
I'd say changing the name would be more consistent to me so we think on these values for ranking purposes more than a field type. 

Otherwise we may be associating them to the `dense_vector` field type.